core_java/twitter/src/main/java/ca/jrvs/apps/example/jsonExample

```diff
@@ -0,0 +1,44 @@
+{
+  "symbol":"AAPL",
+  "companyName":"Apple Inc.",
+  "exchange":"Nasdaq Global Select",
+  "description":"Apple Inc is designs, manufactures and markets mobile communication and media devices and personal computers, and sells a variety of related software, services, accessories, networking solutions and third-party digital content and applications.",
+  "CEO":"Timothy D. Cook",
```

Comment:
remove this if it's not used..
it's not a java file, so it shouldn't be under core_java/twitter/src/main/**java**